### PR TITLE
fix(sync-files): remove extra "--" to prevent npm error

### PR DESCRIPTION
**Pull request by mrMetalWood**

### CODE REVIEW

**Comment by yfr** [comment]
But does the sync task get the --watch switch?

**Comment by Scarysize** [comment]
@yfr No, we need to create two separate npm scripts. `sync-files` fails when the `--watch` option is passed AFTER the directories.

**Comment by mrMetalWood** [comment]
So one task
`"copy-files": "sync-files --no-delete src/static build/static"`

And the other
`"sync-files": "sync-files --watch --no-delete src/static build/static"`

?

**Comment by Scarysize** [comment]
Yes, would probably name `copy-files` & `watch-files`.

File: templates/package.json [hunk 1]
```diff
@@ -11,14 +11,15 @@
...
+    "build": "npm run clean && npm run copy-files && NODE_ENV=production webpack -p",
     "changelog": "conventional-changelog -p angular -i CHANGELOG.md -s -r 0",
     "clean": "rm -rf build/*",
     "deploy": "gcloud-storage-upload -s '{{project-name}}' -p build",
     "postversion": "npm run changelog && git add CHANGELOG.md && git commit --amend --no-edit",
     "preversion": "npm test",
-    "start": "npm run clean && npm run sync-files --watch & webpack-dev-server",
-    "sync-files": "sync-files --no-delete src/static build/static",
+    "start": "npm run clean && npm run watch-files & webpack-dev-server",
+    "copy-files": "sync-files --no-delete src/static build/static",
```
Comment (Scarysize) [inline templates/package.json]: nit pick. Please sort script alphabetically 🙄

**Pull request merged**